tools/adb/disable_talkback.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+# Copyright 2025 New Vector Ltd.
+#
+# SPDX-License-Identifier: AGPL-3.0-only OR LicenseRef-Element-Commercial
+# Please see LICENSE files in the repository root for full details.
+
+adb shell settings put secure enabled_accessibility_services com.android.talkback/com.google.android.marvin.talkback.TalkBackService
```

Comment:
This is like a trick to put a service that does not exist, right?
Maybe change the value to something more explicit like

`adb shell settings put secure enabled_accessibility_services null`

Reply:
Good call, I thought I had tested that before and it hadn't worked, but I retried now and removing the setting works as expected.